superset/cli/thumbnails.py

```diff
@@ -76,7 +76,7 @@ def compute_thumbnails(
     def compute_generic_thumbnail(
         friendly_type: str,
         model_cls: Union[type[Dashboard], type[Slice]],
-        model_id: int,
+        model_id: list[int],
```

Comment:
Nit: For disambiguation and clarity, we may want to call the argument in the inner function `model_ids`.